ksqldb-functional-tests/src/test/resources/query-validation-tests/alter.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
somewhere we should add a test that adds a column with the same name as a column that already exists